updates/33.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE person ADD COLUMN GND varchar(255) DEFAULT NULL;
+
+CREATE INDEX idx_gnd ON person (GND);
```

Comment:
We don't use the idx_ naming schema. just call the Index the same as the column (GND), that's currently the way all other indexes in the person tables are named.